tzrec/datasets/data_parser.py

```diff
@@ -116,6 +116,39 @@ def __init__(
                 logger.info(f"self.user_feats: {self.user_feats}")
                 logger.info(f"self.user_inputs: {self.user_inputs}")
 
+        logger.info(f"self.sparse_keys: {self.sparse_keys}")
+        logger.info(f"self.dense_keys: {self.dense_keys}")
+        logger.info(f"self.sequence_dense_keys: {self.sequence_dense_keys}")
+
+        # get all key.values as the real keys list
+        self.data_list_keys = []
+        self.dense_keys_list = []
+        self.sparse_keys_list = []
+        self.sequence_dense_keys_list = []
+        self.get_data_list_keys()
+
+    def get_data_list_keys(self) -> None:
```

Comment:
missing `.weight`. do not build keys again, use batch.to_dict().keys()

Reply:
ok